diff --git a/base/BUILD.gn b/base/BUILD.gn
index b2d45d2723787e0f5ca542f310ab2b0ef013ff05..14fd63e820076be0e1e67cfd6edc9acb0b5de120 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -5153,6 +5153,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/UserDataHost.java",
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
     ]
+    sources += brave_java_base_sources
 
     resources_package = "org.chromium.base"
 
@@ -5844,6 +5845,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
