diff --git a/build/config/siso/main.star b/build/config/siso/main.star
index ac16471a9c538e735918cc303aad70509d757ea3..ddde20c7249bdbe9c9b15825c432faddcad2a698 100644
--- a/build/config/siso/main.star
+++ b/build/config/siso/main.star
@@ -24,6 +24,7 @@ load("./rust.star", "rust")
 load("./simple.star", "simple")
 load("./typescript_all.star", "typescript_all")
 load("./windows.star", chromium_windows = "chromium")
+load("./brave_siso_config.star", "brave_siso_config")
 
 def __setup_python(ctx, step_config):
     for rule in step_config["rules"]:
@@ -147,6 +148,7 @@ def init(ctx):
     handlers.update(reclient.handlers)
     handlers.update(typescript_all.handlers)
 
+    brave_siso_config.configure(ctx, step_config, filegroups, handlers)
     return module(
         "config",
         step_config = json.encode(step_config),
