diff --git a/chrome/browser/resources/settings/people_page/sync_account_control.ts b/chrome/browser/resources/settings/people_page/sync_account_control.ts
index 92be0e8471bd7f25e20a970f3537e61cbb452a91..f7205fb2d081ec107cb7925c8c40e523b826b7ce 100644
--- a/chrome/browser/resources/settings/people_page/sync_account_control.ts
+++ b/chrome/browser/resources/settings/people_page/sync_account_control.ts
@@ -885,5 +885,3 @@ declare global {
   }
 }
 
-customElements.define(
-    SettingsSyncAccountControlElement.is, SettingsSyncAccountControlElement);
