diff --git a/chrome/browser/signin/signin_promo_util.cc b/chrome/browser/signin/signin_promo_util.cc
index 67c28823e94c1020f7bc9880f6bb632430df6db3..caf686c22cb27cf8f044aa3db22c5867aff9eb5c 100644
--- a/chrome/browser/signin/signin_promo_util.cc
+++ b/chrome/browser/signin/signin_promo_util.cc
@@ -646,6 +646,7 @@ bool CanShowPromoForSyncableDataType(SignInPromoType type, Profile& profile) {
 // Needs additional checks depending on the type of the promo.
 // `profile` is the profile of the tab the promo would be shown on.
 bool ShouldShowSignInPromoCommon(Profile& profile, SignInPromoType type) {
+  SHOULD_SHOW_SIGNIN_PROMO_COMMON
   if (profile.IsOffTheRecord()) {
     return false;
   }
