diff --git a/chrome/browser/ui/browser.h b/chrome/browser/ui/browser.h
index d478c7740e55d4fb4e6ece2e50ed5118389cac30..a1441949ad47b0cb33e2c0c04c5cd2f906249c13 100644
--- a/chrome/browser/ui/browser.h
+++ b/chrome/browser/ui/browser.h
@@ -173,6 +173,8 @@ class Browser : public TabStripModelObserver,
   const DesktopBrowserWindowCapabilities* capabilities() const override;
 
  private:
+  friend class BookmarkPrefsService;
+  friend class BraveBrowser;
   friend BrowserWindowInterface* CreateBrowserWindow(
       BrowserWindowCreateParams create_params);
   friend std::unique_ptr<Browser> DeprecatedCreateOwnedBrowserWindowForTesting(
@@ -218,7 +220,7 @@ class Browser : public TabStripModelObserver,
 
   // Handle changes to tab strip model.
   void OnTabInsertedAt(content::WebContents* contents, int index);
-  void OnTabClosing(tabs::TabInterface* tab, bool* had_active_modal_dialog);
+  virtual void OnTabClosing(tabs::TabInterface* tab, bool* had_active_modal_dialog);
   void OnTabDetached(tabs::TabInterface* tab,
                      bool was_active,
                      bool had_active_modal_dialog);
