diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 295399e36262ef2ebe1ad892d6812ed2ffdb56b2..8681384baa4bfce9bf88f313edb367c0f6a50d61 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -3333,6 +3333,7 @@ source_set("browser") {
       "//chrome/browser/safe_browsing/suspicious_site_warnings",
     ]
   }
+  public_deps += [ "//brave/browser:core" ]
 
   # This target intentionally has no sources of its own and no
   # allow_circular_includes_from; it only aggregates dependencies. Do NOT add
@@ -3502,6 +3503,7 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
