diff --git a/chrome/browser/ui/tabs/features.cc b/chrome/browser/ui/tabs/features.cc
index 024de8a4e2d7a19871e9bf61fea2c7621e725fd3..96ec71dc0c479ebbab8d8c6909fce8657f8d8cc5 100644
--- a/chrome/browser/ui/tabs/features.cc
+++ b/chrome/browser/ui/tabs/features.cc
@@ -26,7 +26,8 @@ BASE_FEATURE(kTabSearchCjkWordBoundary, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kVerticalTabs, base::FEATURE_DISABLED_BY_DEFAULT);
 
-BASE_FEATURE(kVerticalTabsLaunch, base::FEATURE_ENABLED_BY_DEFAULT);
+// kVerticalTabsLaunch feature state is enforced via plaster rewrite.
+BASE_FEATURE(kVerticalTabsLaunch, base::FEATURE_DISABLED_BY_DEFAULT);
 BASE_FEATURE_PARAM(bool,
                    kVerticalTabsToggleInTabContextMenu,
                    &kVerticalTabsLaunch,
