diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 076f2815c40a4c24548b678653b10b81ace38597..cd6d5e75da2fac9c2c1faf3a6544b2f05b71fbba 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4953,6 +4953,7 @@ constexpr char kEnterpriseIsolatedModeInternalName[] =
 
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
