diff --git a/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc b/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
index 642d88bdf689de29b3db5c7000ebdafd6794eb3f..7017e05210a45e9a9a11a42f2bf4da3a6810c330 100644
--- a/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
+++ b/chrome/browser/ui/webui/tab_search/tab_search_page_handler_browsertest.cc
@@ -101,6 +101,8 @@ class MockPage : public tab_search::mojom::Page {
   MOCK_METHOD(void, TabUpdated, (tab_search::mojom::TabUpdateInfoPtr));
   MOCK_METHOD(void, TabsRemoved, (tab_search::mojom::TabsRemovedInfoPtr));
   MOCK_METHOD(void, TabUnsplit, ());
+  MOCK_METHOD(void, TabFocusShowFREChanged, (bool show));
+  MOCK_METHOD(void, TabFocusEnabledChanged, (bool enabled));
 };
 
 void ExpectNewTab(const tab_search::mojom::Tab* tab,
