diff --git a/chrome/browser/resources/settings/people_page/sync_controls.ts b/chrome/browser/resources/settings/people_page/sync_controls.ts
index cd299c7b2d4f42db8f59608c9e89485d68f40c69..35aa008a6f072b00195bf38d45ed11c267dbc8fb 100644
--- a/chrome/browser/resources/settings/people_page/sync_controls.ts
+++ b/chrome/browser/resources/settings/people_page/sync_controls.ts
@@ -143,7 +143,7 @@ export class SettingsSyncControlsElement extends
 
     const router = Router.getInstance();
     const currentRoute = router.getCurrentRoute();
-    if (currentRoute === routes.SYNC_ADVANCED) {
+    if (currentRoute === routes.BRAVE_SYNC_SETUP) {
       this.syncBrowserProxy_.didNavigateToSyncPage();
     }
     if (loadTimeData.getBoolean('replaceSyncPromosWithSignInPromos') &&
@@ -375,6 +375,7 @@ export class SettingsSyncControlsElement extends
 
   protected disableTypeCheckBox_(dataTypeManaged: boolean|undefined|null):
       boolean {
+    if (dataTypeManaged) { return true; } if (!this.syncStatus || !this.syncPrefs) { return !!this.syncPrefs?.syncAllDataTypes; }
     if (!this.syncStatus) {
       return true;
     }
@@ -384,7 +385,7 @@ export class SettingsSyncControlsElement extends
     }
 
     if (this.syncStatus.signedInState === SignedInState.SYNCING) {
-      return !!this.syncPrefs?.syncAllDataTypes;
+      return this.syncPrefs?.syncAllDataTypes;
     }
 
     // Toggles should be disabled on the account settings page if sync is
