diff --git a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
index f983d7e5adda9c7e1bc5c52a23eb672130968565..3df0b3e29865de2aa800fe65d07e988f2adab819 100644
--- a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
+++ b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
@@ -76,7 +76,7 @@ content::WebUIDataSource* CreateAndAddBookmarksUIHTMLSource(Profile* profile) {
       {"editDialogInvalidUrl", IDS_BOOKMARK_MANAGER_INVALID_URL},
       {"editDialogNameInput", IDS_BOOKMARK_MANAGER_NAME_INPUT_PLACE_HOLDER},
       {"editDialogUrlInput", IDS_BOOKMARK_MANAGER_URL_INPUT_PLACE_HOLDER},
-      {"emptyList", IDS_BOOKMARK_MANAGER_EMPTY_LIST},
+      {"emptyList", IDS_BRAVE_BOOKMARK_MANAGER_EMPTY_LIST},
       {"emptyUnmodifiableList", IDS_BOOKMARK_MANAGER_EMPTY_UNMODIFIABLE_LIST},
       {"folderLabel", IDS_BOOKMARK_MANAGER_FOLDER_LABEL},
       {"importBegan", IDS_BOOKMARK_MANAGER_MENU_IMPORT_BEGAN},
