diff --git a/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc b/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc
index 77b24a27d2c044f90f1f249d3513f3a93ba99a83..741b7d4719326c4185d389d7c4fd2e612ab4f182 100644
--- a/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc
+++ b/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc
@@ -721,6 +721,7 @@ void DownloadToolbarUIController::UpdateIcon() {
   GetImageBadge(browser_view_)
       ->UpdateImage(is_active, progress_download_count, progress_color,
                     badge_background_color);
+  UpdateIcon_BraveImpl(browser_view_, *new_icon, state_, active_, action_item_);
 
   // Do not show the progress ring when there is no in progress download.
   if (state_ == IconState::kComplete || progress_info_.download_count == 0) {
