diff --git a/chrome/common/chrome_content_client.cc b/chrome/common/chrome_content_client.cc
index 1136dad871e5fbc2389189715f5b15429f8c9fa6..dd36acea472edee7b86a8738a92d1aba12621ae0 100644
--- a/chrome/common/chrome_content_client.cc
+++ b/chrome/common/chrome_content_client.cc
@@ -122,6 +122,7 @@ void ChromeContentClient::SetGpuInfo(const gpu::GPUInfo& gpu_info) {
 
 void ChromeContentClient::AddPlugins(
     std::vector<content::WebPluginInfo>* plugins) {
+  [&]() -> void {
 #if BUILDFLAG(ENABLE_PDF)
 #if BUILDFLAG(GOOGLE_CHROME_BRANDING)
   static constexpr char16_t kPDFPluginName[] = u"Chrome PDF Plugin";
@@ -144,6 +145,8 @@ void ChromeContentClient::AddPlugins(
   pdf_info.type = content::WebPluginInfo::PLUGIN_TYPE_BROWSER_INTERNAL_PLUGIN;
   plugins->push_back(pdf_info);
 #endif  // BUILDFLAG(ENABLE_PDF)
+  }();
+  RenameChromiumPdfPluginToChrome(plugins);
 }
 
 void ChromeContentClient::AddContentDecryptionModules(
