diff --git a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
index 1ebc539f8db8e8676a7a580b1033e131fd2f73c1..56d0d5e2207953c6318c7719a4c76b37405b68f2 100644
--- a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
+++ b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
@@ -1268,4 +1268,4 @@ declare global {
   }
 }
 
-customElements.define(PowerBookmarksListElement.is, PowerBookmarksListElement);
+// customElements.define(PowerBookmarksListElement.is, PowerBookmarksListElement);
