diff --git a/chrome/android/chrome_java_sources.gni b/chrome/android/chrome_java_sources.gni
index 75390c457b5d44c17a918cf84207b10c85646da4..2a05a801fc458203e9e29873089cb9b5613ccced 100644
--- a/chrome/android/chrome_java_sources.gni
+++ b/chrome/android/chrome_java_sources.gni
@@ -1392,3 +1392,4 @@ if (enable_screen_capture) {
     "java/src/org/chromium/chrome/browser/media/TabSharingUIManager.java",
   ]
 }
+chrome_java_sources -= ["java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPageView.java",]
