diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ntp/RecentTabsCoordinator.java b/chrome/android/java/src/org/chromium/chrome/browser/ntp/RecentTabsCoordinator.java
index f0376cbe18a65ef9dbbbc4c4e0b751b3d5fc5b5f..55105ecb387be67ae5ac1e5966fcb9121ed56821 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/ntp/RecentTabsCoordinator.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/ntp/RecentTabsCoordinator.java
@@ -79,7 +79,7 @@ public class RecentTabsCoordinator
         LayoutInflater inflater = LayoutInflater.from(activity);
 
         mContextMenuManager =
-                new ContextMenuManager(
+                new org.chromium.chrome.browser.native_page.BraveContextMenuManager(
                         navigationDelegate, this, mActivity::closeContextMenu, "RecentTabs");
 
         mView = (ViewGroup) inflater.inflate(R.layout.recent_tabs_page, parent, false);
