diff --git a/chrome/browser/ui/autofill/chrome_autofill_client.cc b/chrome/browser/ui/autofill/chrome_autofill_client.cc
index e42da1e50f4f4fb92240a55c09be27737bfc6d0a..2ebf28605557e83f3ad5cc6e3a348509f1f4e838 100644
--- a/chrome/browser/ui/autofill/chrome_autofill_client.cc
+++ b/chrome/browser/ui/autofill/chrome_autofill_client.cc
@@ -285,6 +285,12 @@ ui::ElementIdentifier GetElementId(AutofillClient::IphFeature iph_feature) {
 // static
 void ChromeAutofillClient::CreateForWebContents(
     content::WebContents* web_contents) {
+  if (!FromWebContents(web_contents)) {
+    web_contents->SetUserData(
+        UserDataKey(),
+        base::WrapUnique(new BraveChromeAutofillClient(web_contents)));
+  }
+  return;
   DCHECK(web_contents);
   if (!FromWebContents(web_contents)) {
     web_contents->SetUserData(
