diff --git a/chrome/browser/ui/views/side_panel/BUILD.gn b/chrome/browser/ui/views/side_panel/BUILD.gn
index a21ef8e1664983c5195d612403314ee7baf640f6..f3d8d95b9edf742764d5c5d106cc87aafbd62c16 100644
--- a/chrome/browser/ui/views/side_panel/BUILD.gn
+++ b/chrome/browser/ui/views/side_panel/BUILD.gn
@@ -179,6 +179,7 @@ source_set("side_panel") {
     "//ui/webui",
   ]
 
+  import("//brave/browser/ui/views/side_panel/sources.gni") deps += brave_browser_ui_views_side_panel_deps public_deps += brave_browser_ui_views_side_panel_public_deps
   if (enable_extensions) {
     public_deps += [
       "//extensions/browser",
