diff --git a/chrome/browser/resources/bookmarks/command_manager.ts b/chrome/browser/resources/bookmarks/command_manager.ts
index 01bd612f0d196a381e73ae108528c4173f0a7ea9..4f5113a556cd774f6776f47a44ef57043cdebff7 100644
--- a/chrome/browser/resources/bookmarks/command_manager.ts
+++ b/chrome/browser/resources/bookmarks/command_manager.ts
@@ -482,7 +482,7 @@ export class BookmarksCommandManagerElement extends
         chrome.bookmarkManagerPrivate.export();
         break;
       case Command.HELP_CENTER:
-        window.open('https://support.google.com/chrome/?p=bookmarks');
+        window.open('https://community.brave.app', undefined, 'noreferrer');
         break;
       case Command.OPEN_BOOKMARK:
       case Command.OPEN_FOLDER:
