diff --git a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
index d6df10f70e1ff75c6a40d06af72b5067a4534eb0..e2c8d5bd3024e28a354f58754bca3ea8f1d7cb59 100644
--- a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
+++ b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
@@ -525,6 +525,7 @@ class AppMenu implements OnKeyListener {
             }
         }
 
+        if (this instanceof BraveAppMenu) ((BraveAppMenu) this).updatePopup(popup, isByPermanentButton);
         int popupHeight = calculateMenuHeight();
         popup.setHeight(popupHeight);
 
