diff --git a/chrome/browser/ui/tab_helpers.cc b/chrome/browser/ui/tab_helpers.cc
index e45f831fe5ad66b682bad1d4f8cb1c4018404f8b..ee1626f6bc705028b9c28801dbadf5c780ab0fae 100644
--- a/chrome/browser/ui/tab_helpers.cc
+++ b/chrome/browser/ui/tab_helpers.cc
@@ -854,4 +854,5 @@ void TabHelpers::AttachTabHelpers(WebContents* web_contents,
 
   // This is common code for all of us. PLEASE DO YOUR PART to keep it tidy and
   // organized.
+  BRAVE_TAB_HELPERS
 }
