diff --git a/components/history/core/browser/BUILD.gn b/components/history/core/browser/BUILD.gn
index 7d182e866467d4235f896719a8779e98f76311e2..012993040ed1caa34d3bbe34707545558b07c75e 100644
--- a/components/history/core/browser/BUILD.gn
+++ b/components/history/core/browser/BUILD.gn
@@ -156,6 +156,7 @@ static_library("browser") {
     "//ui/gfx",
     "//url",
   ]
+  import("//brave/components/history/core/browser/sources.gni") sources += brave_components_history_core_browser_sources deps += brave_components_history_core_browser_deps
 }
 
 source_set("unit_tests") {
