diff --git a/components/sync/engine/cycle/data_type_tracker.cc b/components/sync/engine/cycle/data_type_tracker.cc
index 30ab8020b2471cd75526254439e7cdd0e64a0d5f..79271ecfdb559d86203157ee16cc5d2b8cd1cd12 100644
--- a/components/sync/engine/cycle/data_type_tracker.cc
+++ b/components/sync/engine/cycle/data_type_tracker.cc
@@ -121,6 +121,7 @@ base::TimeDelta GetDefaultLocalChangeNudgeDelay(DataType data_type) {
     case THEMES_ANDROID:
     case NOTEBOOK:
     case JOURNEY:
+    case AI_CHAT_CONVERSATION:
       return kMediumLocalChangeNudgeDelay;
     case UNSPECIFIED:
       NOTREACHED();
@@ -201,6 +202,7 @@ bool CanGetCommitsFromExtensions(DataType data_type) {
     case THEMES_ANDROID:
     case NOTEBOOK:
     case JOURNEY:
+    case AI_CHAT_CONVERSATION:
       return false;
     case UNSPECIFIED:
       NOTREACHED();
