diff --git a/content/browser/web_contents/web_contents_impl.h b/content/browser/web_contents/web_contents_impl.h
index eaace9aa26f3617e00fc0d833c9e260def6718bd..09654dda896c534b53bb4414d507d3713220a43b 100644
--- a/content/browser/web_contents/web_contents_impl.h
+++ b/content/browser/web_contents/web_contents_impl.h
@@ -1702,6 +1702,8 @@ class CONTENT_EXPORT WebContentsImpl
   // base::trace_event::TraceSessionObserver implementation:
   void OnStart(const perfetto::DataSourceBase::StartArgs&) override;
 
+  bool GetShouldDoLearningForTesting() override;
+
  private:
   using FrameTreeIterationCallback = base::FunctionRef<void(FrameTree&)>;
   using RenderViewHostIterationCallback =
