diff --git a/content/renderer/render_frame_impl.cc b/content/renderer/render_frame_impl.cc
index f52f4d399f2524a80b184932a1c6e97596c5fa4b..3c61df58a811912e3b00e1c31dad2d7bb4b8a069 100644
--- a/content/renderer/render_frame_impl.cc
+++ b/content/renderer/render_frame_impl.cc
@@ -5035,6 +5035,7 @@ bool RenderFrameImpl::ShouldUseUserAgentOverride() const {
   // Temporarily return early and fix properly as part of
   // https://crbug.com/426555.
   if (web_view->MainFrame()->IsWebRemoteFrame()) {
+    BRAVE_RENDER_FRAME_IMPL_SHOULD_USE_USER_AGENT_OVERRIDE
     return false;
   }
   const WebLocalFrame* main_frame = web_view->MainFrame()->ToWebLocalFrame();
