diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index ec2fa165a183e0f13bb22633a8f97f1aa8d2b588..6b0f380260a1aed1baee29dbccfe30869231ae9e 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3614,6 +3614,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
