diff --git a/content/browser/hid/hid_service.cc b/content/browser/hid/hid_service.cc
index 913db77cd6cfff15d809ae9f1d7407f1b32f92f5..06257ca7049cab7637b6d6198cb9061cab295df1 100644
--- a/content/browser/hid/hid_service.cc
+++ b/content/browser/hid/hid_service.cc
@@ -436,11 +436,13 @@ void HidService::RequestDevice(
     RequestDeviceCallback callback) {
   HidDelegate* delegate = GetContentClient()->browser()->GetHidDelegate();
   if (!render_frame_host_ ||
-      !FrameTreeNode::From(render_frame_host_)
+      (!delegate->AllowRequestDeviceWithoutTransientActivation(
+           render_frame_host_) &&
+       !FrameTreeNode::From(render_frame_host_)
            ->UpdateUserActivationState(
                blink::mojom::UserActivationUpdateType::
                    kConsumeTransientActivation,
-               blink::mojom::UserActivationNotificationType::kNone) ||
+               blink::mojom::UserActivationNotificationType::kNone)) ||
       !delegate->CanRequestDevicePermission(GetBrowserContext(), origin_)) {
     std::move(callback).Run(std::vector<device::mojom::HidDeviceInfoPtr>());
     return;
