diff --git a/crypto/unexportable_key_metrics.cc b/crypto/unexportable_key_metrics.cc
index 4dff13d0e1c57661e93c44c0288ad40d9941ca56..94bf321a20283e19bc930a13f60773a97833d2c0 100644
--- a/crypto/unexportable_key_metrics.cc
+++ b/crypto/unexportable_key_metrics.cc
@@ -30,6 +30,7 @@ enum class KeyType {
 const SignatureVerifier::SignatureAlgorithm kAllAlgorithms[] = {
     SignatureVerifier::SignatureAlgorithm::ECDSA_SHA256,
     SignatureVerifier::SignatureAlgorithm::RSA_PKCS1_SHA256,
+    SignatureVerifier::SignatureAlgorithm::ECDSA_SHA384,
 };
 
 constexpr char kTestKeyName[] = "ChromeMetricsTestKey";
@@ -147,6 +148,7 @@ internal::TPMSupport MeasureVirtualTpmOperations() {
   if (algo) {
     switch (*algo) {
       case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA256:
+      case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA384:
         supported_virtual_algo = internal::TPMSupport::kECDSA;
         break;
       case SignatureVerifier::SignatureAlgorithm::RSA_PKCS1_SHA256:
@@ -218,6 +220,7 @@ void MeasureTpmOperationsInternal(UnexportableKeyProvider::Config config) {
   if (algo) {
     switch (*algo) {
       case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA256:
+      case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA384:
         supported_algo = internal::TPMSupport::kECDSA;
         break;
       case SignatureVerifier::SignatureAlgorithm::RSA_PKCS1_SHA256:
@@ -385,6 +388,8 @@ std::string AlgorithmToString(SignatureVerifier::SignatureAlgorithm algorithm) {
       return "RSA";
     case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA256:
       return "ECDSA";
+    case SignatureVerifier::SignatureAlgorithm::ECDSA_SHA384:
+      return "ECDSA";
   }
 }
 
