diff --git a/DEPS b/DEPS
index 0c3b285973e19c2b29c25d57a3e924aed7bca5c3..30d66dd34f2a37f325269fe830ea8aec7f9e86c8 100644
--- a/DEPS
+++ b/DEPS
@@ -4524,7 +4524,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
