diff --git a/net/base/features.cc b/net/base/features.cc
index e5eba75a78846cbd50807c92711e5c33c5b32bb3..28c2ef74e04a1c336f2c36d28739bbc81b3767db 100644
--- a/net/base/features.cc
+++ b/net/base/features.cc
@@ -210,8 +210,9 @@ BASE_FEATURE(kSplitCacheByNetworkIsolationKey,
 BASE_FEATURE(kSplitCodeCacheByNetworkIsolationKey,
              base::FEATURE_DISABLED_BY_DEFAULT);
 
+// kPartitionConnectionsByNetworkIsolationKey feature state is enforced via plaster rewrite.
 BASE_FEATURE(kPartitionConnectionsByNetworkIsolationKey,
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 BASE_FEATURE(kSplitHostCacheByNetworkAnonymizationKey,
              base::FEATURE_ENABLED_BY_DEFAULT);
@@ -401,6 +402,7 @@ BASE_FEATURE(kForceThirdPartyCookieBlocking,
 
 BASE_FEATURE(kEnableEarlyHintsOnHttp11, base::FEATURE_DISABLED_BY_DEFAULT);
 
+// kEnableWebTransportDraft07 feature state is enforced via plaster rewrite.
 BASE_FEATURE(kEnableWebTransportDraft07, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kWebTransportFineGrainedThrottling,
@@ -712,7 +714,8 @@ BASE_FEATURE(kRestrictAbusePorts, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kRestrictAbusePortsOnLocalhost, base::FEATURE_ENABLED_BY_DEFAULT);
 
-BASE_FEATURE(kTLSTrustAnchorIDs, base::FEATURE_ENABLED_BY_DEFAULT);
+// kTLSTrustAnchorIDs feature state is enforced via plaster rewrite.
+BASE_FEATURE(kTLSTrustAnchorIDs, base::FEATURE_DISABLED_BY_DEFAULT);
 
 BASE_FEATURE(kNonMtcTrustAnchorIDs, base::FEATURE_ENABLED_BY_DEFAULT);
 
