diff --git a/third_party/blink/renderer/bindings/idl_in_modules.gni b/third_party/blink/renderer/bindings/idl_in_modules.gni
index 88e15cce47153fdf0cce3756557d1c18194c09b9..86634f090c5f8c8e8c354afe08e6d4981e7006c9 100644
--- a/third_party/blink/renderer/bindings/idl_in_modules.gni
+++ b/third_party/blink/renderer/bindings/idl_in_modules.gni
@@ -1305,6 +1305,7 @@ static_idl_files_in_modules = [
   "//third_party/blink/renderer/modules/xr/xr_webgl_layer_init.idl",
   "//third_party/blink/renderer/modules/xr/xr_webgl_sub_image.idl",
 ]
+static_idl_files_in_modules += brave_static_idl_files_in_modules static_idl_files_in_modules -= brave_static_idl_files_to_exclude_in_modules
 
 # Direct Sockets
 # This uses target_os rather than current_os (which is what is_android is set
